CHANGELOG.md

```diff
@@ -64,6 +64,7 @@ You can find its changes [documented below](#070---2021-01-01).
 - Scope: expose scoped state using state() and state_mut() ([#2082] by [@rjwittams]
 - Tabs: allow getting and setting the tab index of a Tabs widget ([#2082] by [@rjwittams]
 - `RangeSlider` and `Annotated` ([#1979] by [@xarvic])
+- Add `Checkbox::from_label` constructor ([#2111] by [@maurerdietmar])
```

Comment:
[#2111] should be a link to your PR. There is a list of links at the end of this file.